conf/messages-ecc.cy

```diff
@@ -112,4 +112,9 @@ doe.error.invalid-date=TRANSLATE Date of establishment must be a real date
 
 vat.error.empty-date=TRANSLATE Enter your effective VAT date
 vat.error.future-date=TRANSLATE Effective VAT date must be in the past
-vat.error.invalid-date=TRANSLATE Effective VAT date must be a real date
\ No newline at end of file
+vat.error.invalid-date=TRANSLATE Effective VAT date must be a real date
+
+cds.feedback.continue=TRANSLATE More about [SERVICE]
```

Comment:
So just to confirm if someone define the continue feedback url this will be displayed on the page just an url with this text?